flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -144,11 +150,31 @@
 
 	protected final List<Tuple2<String, DistributedCache.DistributedCacheEntry>> cacheFile = new ArrayList<>();
 
+	private final ExecutorServiceLoader executorServiceLoader;
+
+	private final Configuration configuration;
```

Comment:
How does this configuration relate to `LocalExecutionEnvironment#configuration` ?

Cannot we unify the two?

Reply:
Yes that is the plan for follow-up commits which will use the newly introduced `execute()` method.